README.md

```
@@ -28,5 +28,7 @@ Our platform will facilitate coordination among various volunteer organizations,
 Resources we find useful in our learning journey
 | Title | link |
 | ------ | ------ |
-| TailwindCSS in a project | [https://www.youtube.com/watch?v=tS7upsfuxmo][PlDb] |
-| PostgreSQL Crash Course | [https://youtu.be/SpfIwlAYaKk?feature=shared][PlGh] |
+| TailwindCSS in a project | https://www.youtube.com/watch?v=tS7upsfuxmo |
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)